libs/stache/src/lib/modules/nav/link.directive.spec.ts

```diff
@@ -78,154 +96,152 @@ describe('StacheLinkDirective', () => {
 
     TestBed.configureTestingModule({
       imports: [StacheNavModule],
-      declarations: [
-        StacheRouterLinkTestComponent,
-        StacheRouterLinkTestLocalRouteComponent,
-      ],
+      declarations: [StacheRouterLinkTestComponent],
       providers: [
         LocationStrategy,
         { provide: LocationStrategy, useClass: PathLocationStrategy },
         { provide: APP_BASE_HREF, useValue: '/' },
         { provide: StacheNavService, useValue: mockNavService },
         { provide: StacheRouteService, useValue: mockRouteService },
       ],
-    }).compileComponents();
+    });
 
     fixture = TestBed.createComponent(StacheRouterLinkTestComponent);
     directiveElement = fixture.debugElement.query(
       By.directive(StacheRouterLinkDirective)
     );
-    debugElement = fixture.debugElement;
   });
 
   it('should render the component', () => {
     expect(fixture).toExist();
   });
 
   it('should have a route input', () => {
+    fixture.componentInstance.routerLink = 'test-route';
```

Comment:
👇 Had to rewrite the tests to use an actual fixture since they were accessing private properties of the component. The expectations should be the same, however.